observing/makesdf.py

```diff
@@ -208,6 +218,11 @@ def make_obs_block(obs_id, start_time:str, duration, ra = None, dec = None, obj_
     if dec is not None:
         lines += f"OBS_DEC         %+.9f\n" % (dec)
 
+    if az is not None:
+        lines += f"OBS_STP_C1[1]   {az}\n"
+    if alt is not None:
+        lines += f"OBS_STP_C2[1]   {alt}\n"
+
```

Comment:
If "AZALT" is to be a new mode that supports only a single topocentric pointing I would suggest changing these to "OBS_AZ" and "OBS_ALT" to avoid confusion.

Reply:
Thanks for the feedback here. I think I'll try to move my changes into a non-compatible, but explicit AZALT mode. We can work in STEPPED support via `lsl` later.